Lib9c.Policy/NCStagePolicy.cs

```diff
@@ -77,7 +88,8 @@ public IEnumerable<Transaction> Iterate(BlockChain blockChain, bool filtered = t
 
         public bool Stage(BlockChain blockChain, Transaction transaction)
         {
-            if (_accessControlService != null && _accessControlService.IsAccessDenied(transaction.Signer))
+            var acsTxQuota = _accessControlService?.GetTxQuota(transaction.Signer);
+            if (_accessControlService != null && acsTxQuota == 0)
```

Comment:
The `IAccessControlService` seems to return `int?` so is it enough to use the last condition? `null == 0` is false. (ref: https://stackoverflow.com/a/15777768)

```suggestion
            if (acsTxQuota == 0)
```